charts/maildev/templates/ingress.yaml

```diff
@@ -11,26 +11,30 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
+  ingressClassName: {{ .Values.ingress.spec.ingressClassName | quote}}
```

Comment:
Can set this block conditionnal : if variables is not empty in values.yaml

Reply:
Done